python/gps/algorithm/policy_opt/config_tf.py

```diff
@@ -0,0 +1,23 @@
+from gps.algorithm.policy_opt.tf_model_example import example_tf_network
+import os
+checkpoint_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                               '..', 'policy_opt/tf_checkpoint/policy_checkpoint.ckpt'))
+
+POLICY_OPT_TF = {
```

Comment:
Can you put this in policy_opt/config.py? Generally, we have been only using one config file per directory.